src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(value) {
+      this.result += value;
+    },
```

Comment:
each method function shouldn't know anything about the result property
it should accept 2 arguments, and return some result depending on own functionality